### PR TITLE
Move SIGNALFX_CONVENTION to internal configuration

### PR DESCRIPTION
**Pull request by dszmigielski**
## Why

There is new section for internal configuration (env vars that user should not change).

## What

Moving SIGNALFX_CONVENTION to the internal configuration section

## Tests

N/A

### CODE REVIEW

**Comment by dszmigielski** [comment]
@pjanotti @pellared @RassK  Please let me know if there are any env vars that should be in the new section that you are aware of, so we can move them in one PR.

**Comment by RassK** [comment]
> @pjanotti @pellared @RassK Please let me know if there are any env vars that should be in the new section that you are aware of, so we can move them in one PR.

All AppSec vars, we don't support that AppSec integration, but in the same time we didn't delete that part also.

**Review by Kielek — APPROVED** [review]

**Comment by dszmigielski** [comment]
> > @pjanotti @pellared @RassK Please let me know if there are any env vars that should be in the new section that you are aware of, so we can move them in one PR.
> 
> All AppSec vars, we don't support that AppSec integration, but in the same time we didn't delete that part also.

These are not documented in USAGE.md

File: docs/USAGE.md [hunk 1]
```diff
@@ -58,8 +58,7 @@ Use these environment variables to configure the tracing library:
...
 | `SIGNALFX_DIAGNOSTIC_SOURCE_ENABLED` | Enable to generate troubleshooting logs with the `System.Diagnostics.DiagnosticSource` class. | `true` |
-| `SIGNALFX_DISABLED_INTEGRATIONS` | The integrations you want to disable, if any, separated by a comma. These are the supported integrations: AspNetMvc, AspNetWebApi2, DbCommand, ElasticsearchNet5, ElasticsearchNet6, GraphQL, HttpMessageHandler, IDbCommand, MongoDb, NpgsqlCommand, OpenTracing, ServiceStackRedis, SqlCommand, StackExchangeRedis, Wcf, WebRequest |  |
-| `SIGNALFX_CONVENTION` | Sets the semantic and trace id conventions for the tracer. Available values are: `Datadog` (64bit trace id), `OpenTelemetry` (128 bit trace id). | `OpenTelemetry` |
+| `SIGNALFX_DISABLED_INTEGRATIONS` | The integrations you want to disable, if any, separated by a comma. These are the supported integrations: `AspNetMvc`, `AspNetWebApi2`, `DbCommand`, `ElasticsearchNet5`, `ElasticsearchNet6`, `GraphQL`, `HttpMessageHandler`, `IDbCommand`, `MongoDb`, `NpgsqlCommand`, `OpenTracing`, `ServiceStackRedis`, `SqlCommand`, `StackExchangeRedis`, `Wcf`, `WebRequest` |  |
```
Comment (pjanotti) [inline docs/USAGE.md:61]: 🤔 can we delegate the list of actual instrumentations to some other doc? So we could run some tool to get the list and update the document on each release. The list here will tend to get stale.

Anyway, no need to hold such small update for that.

**Review by pjanotti — APPROVED** [review]
The list of integrations seems out of date, but, anyway we should review and perhaps automate the list before switching to next-ver.

**Pull request merged**